README.md

```diff
@@ -7,51 +7,53 @@ This project was built using only built in web services (ie no NPM packages besi
 
 ### 🎨 CSS
-- It's amazing how much of this was achievable just using CSS (and sometimes some JS)
-  - Changing Font
-  - Light mode/Dark mode (exluding system setting)
-  - Word wrap
-  - `window-contrls-overlay`, contributes to app-like appearance
-  - Fake MICA appearance (https://github.com/yell0wsuit/acrylic-mica-css)
-  - Scroll bar
-- Limitaitons (consideration):
-  - It's just important to be organized and efficient with your CSS (ie taking advantage of CSS variables)
-  - Does required JS to be used in tandem. I achieved a lot using Classmaps/Stylemaps from Lit as well as CSS manipulation via vanilla JS(TS).
+
+A lot of this project was an attempt to nail the CSS to create a near 1:1 replica of the native Notepad. I was amazed but all that was possible with vanilla CSS alone and sometimes powered by a little bit of Typescript. Everything from changing the font, toggle color theme, controlling word wrap in the editor, and using `window-contorls-overlay` to create a more _app-like_ look to the PWA when installed on the OS.
+
+I tried my best to keep the CSS organized and take advantage of CSS variables to make my life easier while working on the appearance of this app. Variables allowed me to keep my colors in check and also made it easy to toggle them when the theme switched. I also relied heavily on the Lit directives "StyleMap" and "ClassMap" which allows you to change styles or toggle classes based on conditions (ie the user selecting different settings).
 
 ## 📈 Where do I see room for growth for PWA?
-- window setting with WCO + built in back button (Windows Specific)
-- Something like MICA would be soooo coool + there is already buzz around this (https://twitter.com/diekus/status/1754911118236041630)
\ No newline at end of file
+
+Two things I feel like I wasn't able to nail how I wanted was
+1. The title bar with `window-controls-overlay` was close but I had to manually add a back button and ensure that the "browser part" stayed the correct color to match each theme. If there was a way to have the collapse of WCO and also the back button from the minimal-ui display, that would be a nice solution to my problem.
+2. I have heard buzz around a [MICA like background](https://twitter.com/diekus/status/1754911118236041630) which would be an amazing addition to the web platform. This is not only just super cool to have in an app, but native applications use this widespread and it would be just another way for PWAs to match up to native applications and even look like them too. The native Notepad uses this in the settings page and I used the guidance of this [github repo](https://github.com/yell0wsuit/acrylic-mica-css) to replicate that as best that I can.
```

Comment:
```suggestion
2. I have heard buzz around a [MICA like background](https://twitter.com/diekus/status/1754911118236041630) which would be an amazing addition to the web platform. This is not only just super cool to have in an app, but native applications use this widespread, and it would be just another way for PWAs to match up to native applications and look even more like them. The native Notepad uses this in the settings page, and I used the guidance of this [GitHub repo](https://github.com/yell0wsuit/acrylic-mica-css) to replicate that as best as I can.
```